src/Sulu/Component/Content/Document/Subscriber/BlameSubscriber.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Subscriber;
+
+use Sulu\Component\Content\Document\Behavior\BlameBehavior;
+use Sulu\Component\DocumentManager\Event\ConfigureOptionsEvent;
+use Sulu\Component\DocumentManager\Event\MetadataLoadEvent;
+use Sulu\Component\DocumentManager\Event\PersistEvent;
+use Sulu\Component\DocumentManager\Events;
+use Sulu\Component\Security\Authentication\UserInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+
+/**
+ * Manages user blame (log who creator the document and who updated it last).
+ */
+class BlameSubscriber implements EventSubscriberInterface
+{
+    const CREATOR = 'creator';
+    const CHANGER = 'changer';
+
+    /**
+     * @var TokenStorage
+     */
+    private $tokenStorage;
+
+    /**
+     * @param TokenStorage $tokenStorage
+     */
+    public function __construct(TokenStorage $tokenStorage = null)
+    {
+        $this->tokenStorage = $tokenStorage;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            Events::CONFIGURE_OPTIONS => 'configureOptions',
+            Events::PERSIST => 'handlePersist',
+            Events::METADATA_LOAD => 'handleMetadataLoad',
+        ];
+    }
+
+    /**
+     * @param ConfigureOptionsEvent $event
+     */
+    public function configureOptions(ConfigureOptionsEvent $event)
+    {
+        $event->getOptions()->setDefaults([
+            'user' => null,
+        ]);
+    }
+
+    /**
+     * Adds the creator and changer to the metadata for persisting.
+     *
+     * @param MetadataLoadEvent $event
+     */
+    public function handleMetadataLoad(MetadataLoadEvent $event)
```

Comment:
Again, I'd use a more descriptive and less technical name. What do we do? For instance, "addCreatorAndChangerToMetadata" is long, but says what this method actually does.